WeatherApp/Views/WeatherTableViewCell.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  TableViewCell.swift
+//  WeatherApp
+//
+//  Created by Lucija Balja on 03/08/2020.
+//  Copyright © 2020 Lucija Balja. All rights reserved.
+//
+
+import UIKit
+
+class WeatherTableViewCell: UITableViewCell {
+    private let containerView = UIView()
+    private let temperatureView = UIView()
+    private let weatherIcon = UIImageView()
+    private let cityLabel = UILabel()
+    private let currentTempLabel = UILabel()
+    private let minTempLabel = UILabel()
+    private let maxTempLabel = UILabel()
+    
+    var weather: CityWeather? {
+        didSet {
+            setLabelsText()
+        }
+    }
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupUI()
+        setupConstraints()
+    }
+    
+    private func setLabelsText() {
+        guard let weatherData = weather else { return }
+        
+        weatherIcon.image = UIImage(systemName: weatherData.icon)
+        cityLabel.text = weatherData.city
+        currentTempLabel.text = "\(weatherData.parameters.currentTemperature)°"
+        minTempLabel.text = "\(weatherData.parameters.minTemperature)°"
+        maxTempLabel.text = "\(weatherData.parameters.maxTemperature)°/"
+    }
+    
+    private func setupUI() {
+        contentView.backgroundColor = .systemBlue
+        layer.backgroundColor = UIColor.clear.cgColor
+        layer.shadowColor = UIColor.black.cgColor
+        layer.shadowOffset = CGSize(width: 0, height: 1.0)
+        layer.shadowOpacity = 0.2
+        layer.shadowRadius = 4.0
+          
+        containerView.layer.cornerRadius = 6.0
+        containerView.layer.masksToBounds = true
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+        
+        weatherIcon.styleView()
+        cityLabel.style(size: 30, textAlignment: .left)
+        currentTempLabel.style(size: 40, textAlignment: .right)
+        maxTempLabel.style(size: 15, textAlignment: .right)
+        minTempLabel.style(size: 15, textAlignment: .left)
+        
+        cityLabel.numberOfLines = 0
+        minTempLabel.textColor =  .systemGray4
+        temperatureView.translatesAutoresizingMaskIntoConstraints = false
+        temperatureView.clipsToBounds = true
+        
+        contentView.addSubview(containerView)
+        contentView.addSubview(temperatureView)
+        contentView.addSubview(weatherIcon)
+        contentView.addSubview(cityLabel)
+        temperatureView.addSubview(currentTempLabel)
+        temperatureView.addSubview(minTempLabel)
+        temperatureView.addSubview(maxTempLabel)
+    }
+    
+    private func setupConstraints() {
+        containerView.topAnchor.constraint(equalTo: self.contentView.topAnchor).isActive = true
```

Comment:
You should do the same thing with recurring constants as you did in WeatherDetailView.